pyproject.toml

```diff
@@ -25,7 +25,8 @@ dependencies = [
     "anyio>=4.5",
     "httpx>=0.27.1",
     "httpx-sse>=0.4",
-    "pydantic>=2.11.0,<3.0.0",
+    "pydantic>=2.11.0,<3.0.0; python_version < '3.14'",
```

Comment:
Missing the classifier above.

Reply:
```
    "Programming Language :: Python :: 3.14",
```

Reply:
I was not going to commit to supporting 3.14 until after we got the tests to pass, but here we go...